lib/matobj2.gd

```diff
@@ -1289,6 +1289,25 @@ DeclareOperation( "^", [ IsVectorObj, IsMatrixObj ] );
 # AddCoeffs
 
 
+#############################################################################
+##
+##  IsEmptyMatrix( <matobj> )
+##
+##  <#GAPDoc Label="MatObj_IsEmptyMatrix">
+##  <ManSection>
+##    <Prop Name="IsEmptyMatrix" Arg='M' Label="for matrices"/>
+##    <Returns>A boolean</Returns>
+##    <Description>
+##      Is <K>true</K> if <A>M</A> either has zero columns or zero rows and 
+##      <K>false</K> otherwise.
+##    </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+DeclareProperty( "IsEmptyMatrix", IsMatrixObj );
+
+
+
```

Comment:
I usually tru not to put more than two empty lines in a row. Not a reason to delay this PR, though :-)

Reply:
Thanks, your right. I changed it. ;)